[Bug] Test when executing benchmark tests fails

I get two errors in benchmark tests when running ` EXECUTE_TESTS_ON=master pytest` in dev branch. Could you please check whether you have these errors, too, and fix them in case of yes?

1)
```
E   FileNotFoundError: [Errno 2] No such file or directory: './tests/other/input_timeseries_Harbor_kW.png'
```

2)
```
E       assert 1 == 0
tests/test_benchmark_simple_scenarios.py:86: AssertionError
```


Checklist to make sure that the bug report ist complete:
- [x] OS: *your_operating_system*, *your_distribution*
     xubuntu 18.4
- [x] Branch: *branch_name*, updated on *update_date*
    dev branch updated today
- [x] If applicable: Attach full error message
[error_msg.txt](https://github.com/rl-institut/mvs_eland/files/4655499/error_msg.txt)

- [ ] If applicable: Share screenshots/images of your problem
- [ ] If applicable: Share used input data

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -39,7 +39,7 @@ tipps for module building, and hint that units in the MVS are not checked (PR #2
 - Tables for `simulating_with_the_mvs.rst`: tables/example_multiple_inputs_energyConversion.csv, tables/example_scalar_as_timeseries_energyConversion.csv
 - Test for the module A1 (#141)
 - Test for the module E3 (#143)
-- Test for the module F0 (#142, #304)
+- Test for the module F0 (#142, #304, #335)
 - Test for the module E0 (#146)
 - Test for module F1 (#157, #297, #284)
 - Benchmark test with only PV and grid (#258)
@@ -109,7 +109,6 @@ tipps for module building, and hint that units in the MVS are not checked (PR #2
 - Logging message for maximumCap value (#310)
 - Create_app function in F0 for standalone execution (#317)
 - Crashing evaluation when `evaluated_period < 365/peak_demand_pricing_periods` (#331) by raising an error
-
 ## [0.2.0] - 2020-03-13
 
 ### Added

--- a/src/B0_data_input_json.py
+++ b/src/B0_data_input_json.py
@@ -1,5 +1,6 @@
 import os
 import json
+import copy
 import numpy as np
 import pandas as pd
 
@@ -146,5 +147,5 @@ def load_json(
 
     # add default value if the field PATHS_TO_PLOTS is not already present
     if PATHS_TO_PLOTS not in dict_values:
-        dict_values.update(DICT_PLOTS)
+        dict_values.update(copy.deepcopy(DICT_PLOTS))
     return dict_values

--- a/tests/test_F0_output.py
+++ b/tests/test_F0_output.py
@@ -9,6 +9,7 @@
 - Store dictionary to Json
 """
 
+import copy
 import os
 import sys
 import shutil
@@ -101,7 +102,7 @@ class TestFileCreation:
                 )
             },
         }
-        dict_scalar_capacities.update(DICT_PLOTS)
+        dict_scalar_capacities.update(copy.deepcopy(DICT_PLOTS))
         show_optimal_capacities = F0.plot_optimized_capacities(dict_scalar_capacities)
         assert show_optimal_capacities is False
 
@@ -119,7 +120,7 @@ class TestFileCreation:
                 )
             },
         }
-        dict_scalar_capacities.update(DICT_PLOTS)
+        dict_scalar_capacities.update(copy.deepcopy(DICT_PLOTS))
         show_optimal_capacities = F0.plot_optimized_capacities(dict_scalar_capacities)
         assert show_optimal_capacities is True
 
@@ -155,7 +156,7 @@ class TestFileCreation:
             "simulation_settings": {"path_output_folder": OUTPUT_PATH},
             "optimizedFlows": {"a_bus": BUS},
         }
-        dict_timeseries_test_one_bus.update(DICT_PLOTS)
+        dict_timeseries_test_one_bus.update(copy.deepcopy(DICT_PLOTS))
         F0.store_timeseries_all_busses_to_excel(dict_timeseries_test_one_bus)
         assert (
             os.path.exists(os.path.join(OUTPUT_PATH, "timeseries_all_busses.xlsx"))
@@ -176,7 +177,8 @@ class TestFileCreation:
             "simulation_settings": {"path_output_folder": OUTPUT_PATH},
             "optimizedFlows": {"a_bus": BUS, "b_bus": BUS},
         }
-        dict_timeseries_test_two_busses.update(DICT_PLOTS)
+        print(DICT_PLOTS)
+        dict_timeseries_test_two_busses.update(copy.deepcopy(DICT_PLOTS))
         F0.store_timeseries_all_busses_to_excel(dict_timeseries_test_two_busses)
         assert (
             os.path.exists(os.path.join(OUTPUT_PATH, "timeseries_all_busses.xlsx"))

--- a/tests/test_F0_output.py
+++ b/tests/test_F0_output.py
@@ -197,6 +197,18 @@ class TestFileCreation:
         F0.store_as_json(JSON_TEST_DICTIONARY, OUTPUT_PATH, file_name)
         assert os.path.exists(os.path.join(OUTPUT_PATH, file_name + ".json")) is True
 
+    def teardown_method(self):
+        """ """
+        if os.path.exists(OUTPUT_PATH):
+            shutil.rmtree(OUTPUT_PATH, ignore_errors=True)
+
+
+class TestPDFReportCreation:
+    def setup_method(self):
+        """ """
+        if os.path.exists(OUTPUT_PATH):
+            shutil.rmtree(OUTPUT_PATH, ignore_errors=True)
+
     @pytest.mark.skipif(
         EXECUTE_TESTS_ON not in (TESTS_ON_MASTER),
         reason="Benchmark test deactivated, set env variable "
@@ -204,7 +216,7 @@ class TestFileCreation:
     )
     @mock.patch(
         "argparse.ArgumentParser.parse_args",
-        return_value=PARSER.parse_args(["-o", OUTPUT_PATH, "-f", "-pdf"]),
+        return_value=PARSER.parse_args(["-o", OUTPUT_PATH, "-pdf"]),
     )
     def test_generate_pdf_report(self, m_args):
         """Run the simulation with -pdf option to make sure the pdf file is generated """

